src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -31,6 +31,10 @@
     private static final Pattern GIT_REPOSITORY_LOCATION_PATTERN =
             Pattern.compile("^.*github.com\\/(?<org>.+?)\\/(?<repoName>.+?)\\.git$");
     private static final String DEFAULT_BRANCH = "master";
+    private static final String COMMIT_HASH_REGEX = "^[0-9a-f]*$";
```

Comment:
hmm, your commits matching uses startsWith. So if someone puts empty string into ignore commits, it will basically ignores everything?

maybe should change this to `+` instead.